R/oauth-token.r

```diff
@@ -211,7 +211,12 @@ Token2.0 <- R6::R6Class("Token2.0", inherit = Token, list(
   refresh = function() {
     self$credentials <- refresh_oauth2.0(self$endpoint, self$app,
         self$credentials, self$params$user_params)
-    self$cache()
```

Comment:
Yeah, I think the warning should probably occur in `refresh_oauth2.0` in thinking about it.